pages/page2.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Food Showcase</title>
+
+    <link rel="stylesheet" href="../bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../css/style.css">
+
+    <script
+    src="http://code.jquery.com/jquery-3.3.1.min.js"
+    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+    crossorigin="anonymous"></script>
+    <script src="../bootstrap/js/bootstrap.min.js"></script>
+
+  </head>
+
+  <body>
+
+    <!-- Header with nav and jumbotron -->
+    <header class="container-fluid">
+
+      <!-- Nav-Bar -->
+      <nav class="navbar navbar-inverse navbar-fixed-top">
+        <div class="container">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="https://www.spartaglobal.com">
+              <img src="../images/SG-logo-white.png" alt="Sparta Global">
```

Comment:
I understand your reasoning including this, as it's is a professional piece of work from Sparta, but from a UX perspective I don't think the logo really belongs in the nav bar - it doesn't really relate to what your site is about.